specification/cognitiveservices/data-plane/TextAnalytics/preview/v3.0-Preview.1/TextAnalytics.json

```diff
@@ -0,0 +1,1496 @@
+        }        
+      },
+      "description" : "Contains an input document to be analyzed by the service."
+    },
+    "DocumentError": {
+      "type": "object",
+      "required" : [
+        "id",
+        "error"
+      ],
+      "properties": {
+        "id": {
+          "type": "string",
+          "description" : "Document Id."
+        },
+        "error": {
+          "type": "object",
+          "description" : "Document Error.",
+          "items" : {
+            "$ref" : "#/definitions/Error"
+          }          
+        }
+      }
+    },
+    "Error": {
+      "type": "object",
+      "required" : [
+        "code",
+        "message"
+      ],
+      "properties": {
+        "code": {
+          "type": "string",
+          "description" : "Error code."
+        },
+        "message": {
+          "type": "string",
+          "description" : "Error message."
+        },
+        "target": {
+          "type": "string",
+          "description" : "Error target."
+        },
+        "innererror": {
+          "$ref": "#/definitions/InnerError",
+          "description" : "Inner error contains more specific information."
+        },
+        "details": {
+          "type": "array",
+          "description" : "Details about specific errors that led to this reported error.",
+          "items" : {
+            "$ref" : "#/definitions/Error"
+          }
+        }
+      }
+    },
+    "InnerError": {
+      "type": "object",
+      "required" : [
+        "code",
+        "message"
+      ],
+      "properties": {
+        "code": {
+          "type": "string",
+          "description" : "Error code."
+        },
+        "message": {
+          "type": "string",
+          "description" : "Error message."
+        },
+        "target": {
+          "type": "string",
+          "description" : "Error target."
+        },
+        "innererror": {
+          "$ref": "#/definitions/InnerError",
+          "description" : "Innererror contains more specific information."
+        }
+      }
+    },
+    "SentimentResponse" : {
+      "type" : "object",
+      "required" : [
+        "documents",
+        "errors"
+      ],
+      "properties" : {
+        "documents" : {
+          "type" : "array",
+          "description" : "Sentiment analysis per document.",
+          "items" : {
+            "$ref" : "#/definitions/DocumentSentiment"
+          }
+        },
+        "errors" : {
+          "type" : "array",
+          "description" : "Errors by document id.",
+          "items" : {
+            "$ref" : "#/definitions/DocumentError"
+          }
+        },
+        "statistics" : {
+          "$ref" : "#/definitions/RequestStatistics"
+        },
+        "model-version" : {
+          "type" : "string",
+          "description" : "if model-version=true was specified in the request this field will indicate which model is used for scoring."
```

Comment:
model-version is not set to true..

Reply:
Also, we should always return this, regardless if the value was specified or not in the input.